### DISCUSSION TITLE
Cashflow Quadrant

### DISCUSSION THREAD

**Discussion by trainfarb** [hu3agency/ep0_daoTeaching#54] [#b0mission]
Quotes from Cashflow Quadrant by Robert T. Kiyosaki. After reading the preface, really excited to see how we can apply this guide to financial freedom into a guide for Institutional freedom.

We'll be sharing some of our favorites to on the polymathProcess Instagram account.

**Comment by trainfarb** [comment]
So is this the pursuit of happiness everyone's so jazzed about?

> **To me, this is the shortcoming of traditional education. Millions of people leave school, only to be trapped in jobs they do not like. They know something is missing in life. Many are also trapped financially, earning just enough to survive, wanting to earn more but not knowing what to do.**

_Cashflow Quadrant: Guide to Financial Freedom_ by **Robert T. Kiyosaki**

**Comment by trainfarb** [comment]
### "I mention emotional and spiritual development because that is what it takes to make a permanent change in life."

Cashflow Quadrant: Guide to Financial Freedom by Robert T. Kiyosaki

---
What if we mandated emotional and spiritual development for our institutions?

**Comment by trainfarb** [comment]
### "We are in the Information Age which offers more opportunities for financial reward than ever before."

Cashflow Quadrant: Guide to Financial Freedom by Robert T. Kiyosaki

---
Can we redesign our institutions to offer more opportunities that align with our values?

**Comment by trainfarb** [comment]
### "For people in this group, money isn't the most important thing about their work. Their independence, the freedom to do things their way, and being respect as experts in their field are much more important than money."

Cashflow Quadrant: Guide to Financial Freedom by Robert T. Kiyosaki

---
How many of us are experts in the field of being ourselves?

**Comment by trainfarb** [comment]
### "To be successful as a B[usiness owner] requires:
### 1. Ownership or control of systems, and 
### 2. The ability to lead people. "

Cashflow Quadrant: Guide to Financial Freedom by Robert T. Kiyosaki

---
Can people reach their potential if we create a society that doesn't teach everyone about their opportunities?

**Comment by trainfarb** [comment]
### "The I[nvestor] quadrant is the playground of the rich. Regardless of which quadrant people make their money in, if they hope someday to be rich, they ultimately must come to the I[nvestor] quadrant. It's in the I[nvestor] quadrant that money becomes converted to wealth."

Cashflow Quadrant: Guide to Financial Freedom by Robert T. Kiyosaki

---
Why don't our institutions teach us about true wealth? Is it wise to blindly trust them?

**Comment by trainfarb** [comment]
### "People who take risks change the world. Few people ever got rich without taking risks. Too many people have come to depend on government to eliminate the risks of life."

Cashflow Quadrant: Guide to Financial Freedom by Robert T. Kiyosaki

---
Who coined the term "stay safe" in 2020? Why did everyone fall in line with the phrase without question? Does our safety benefit us or our institutions?

**Comment by trainfarb** [comment]
#### "The beginning of the Information Age should have ushered in an era of smaller government because big government is frankly too expensive to maintain. But rather than becoming savvy, educated investors, millions of people around the world have bought into the archaic ideas of entitlements and pensions for life. This simply isn't feasible."

Cashflow Quadrant: Guide to Financial Freedom by Robert T. Kiyosaki

---
Should we keep giving our money to institutions who don't seem to manage it well? Does it make sense to trust the experts whenever we don't want to think about challenging topics?

**Comment by trainfarb** [comment]
### "The Information Age means we all need to become more self-sufficient, grow up and take personal responsibility for our retirement."

Cashflow Quadrant: Guide to Financial Freedom by Robert T. Kiyosaki

---
Are our institutions meant to act as our parents or children? Should we step into our roles as adults and start collectively leading our institutions?

**Comment by trainfarb** [comment]
### "One of the problems created by the Industrial Age is that too many people are dependent upon the government to solve their individual problems. Today we're facing even bigger problems because we have delegated our personal financial responsibility to the government."

Cashflow Quadrant: Guide to Financial Freedom by Robert T. Kiyosaki

---
What else are we delegating to our institutions? Is it healthy for adults to be dependent on strangers?

**Comment by trainfarb** [comment]
### "Today we all need to be concerned with more than just job security. I think we must also focus on our long-term financial security and not leave that responsibility to a company or government."

Cashflow Quadrant: Guide to Financial Freedom by Robert T. Kiyosaki

---
What is humanity's collective responsibility?

**Comment by trainfarb** [comment]
### "To be successful, you need to learn to overcome your fear of being rejected and to top worrying about what other people say and think about you."

Cashflow Quadrant: Guide to Financial Freedom by Robert T. Kiyosaki

---
What do we gain from playing it safe?

**Comment by trainfarb** [comment]
### "Tragically, most people do not think learning to invest is important. This is why most people believe investing is risky and turn their money over to "experts," most of whom are not really investors, but sales people who make money whether the investor makes money or loses money."

Cashflow Quadrant: Guide to Financial Freedom by Robert T. Kiyosaki

---
Why do we export our thinking to experts/strangers who have nothing to lose if we fail or get hurt? Where else do we do this as a society?

**Reply by Patrick-T-Anderson** [reply]
The [origin of the RETURN](https://en.wikipedia.org/wiki/Theory_of_value_(economics)) remains unknown,
but when you own your home, RETURN is 0!

Rent takes the most of most people's income,
but if we use property properly, there is none!
